app/components/ontology_browse_card_component/ontology_browse_card_component.html.haml

```diff
@@ -3,9 +3,11 @@
     .d-flex
       .browse-ontology-description
         .browse-ontology-title-bar
-          %a.browse-ontology-title{:href => "/ontologies/#{ontology[:acronym]}", data: {'turbo': 'false'}}
+          %a.browse-ontology-title{:href => onto_link, data: {'turbo': 'false'} , style: style_text}
             = ontology[:name]+" ("+ontology[:acronym]+")"
             = private_ontology_icon(ontology[:private])
+            - if external_ontology?
+              = render Display::InfoTooltipComponent.new(text: "Federated ontology from #{onto_link}", icon: 'external-link.svg')
```

Comment:
I don't really like this 'Federated' word, I don't think the regular user will understand what that is (To be discussed in Rome)